pkgs/tools/networking/netproc/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "netproc";
+  version = "unstable-2022-02-11";
```

Comment:
Why not https://github.com/berghetti/netproc/releases/tag/0.6.3 ?

Reply:
The upstream mater branch makes a series of patches to fix build problem, especially ncursesw6, it's difficult to use fetchpatch to apply cleanly without hunk FAILED.